get-started/get-started.css

```diff
@@ -0,0 +1,90 @@
+@import url('https://fonts.googleapis.com/css2?family=Fredoka&display=swap');
+
+:root{
+    --primary-color: #34D399;
+    --secondary-color: ;
```

Comment:
Same thing as mentioned in previous PR, either give color to it or remove that line.